eslint.config.ts

```diff
@@ -3,18 +3,23 @@ import { FlatCompat } from '@eslint/eslintrc';
 import tseslint from 'typescript-eslint';
 import importPlugin from 'eslint-plugin-import';
 import react from 'eslint-plugin-react';
+import unicorn from 'eslint-plugin-unicorn';
 
 const compat = new FlatCompat();
 
 export default tseslint.config(
-  { ignores: ['**/dist/', 'packages/create-waku/template/'] },
+  {
+    ignores: ['**/dist/', 'packages/create-waku/template/', '**/.cache/'],
+  },
   eslint.configs.recommended,
   tseslint.configs.recommended,
   importPlugin.flatConfigs.recommended,
   react.configs.flat.recommended,
   react.configs.flat['jsx-runtime'],
   ...compat.extends('plugin:react-hooks/recommended'),
   {
+    files: ['**/*.{ts,tsx,js,jsx}'],
+    ignores: ['**/dist/', 'packages/create-waku/template/', '.cache/'],
```

Comment:
Do we need this? At least, the second line seems duplicated.

Reply:
good point, changed